src/main/java/rife/continuations/instrument/TypesMethodVisitor.java

```diff
@@ -298,205 +296,107 @@ public void visitInsn(int opcode) {
 
         if (currentNode_ != null) {
             switch (opcode) {
-                case RETURN:
```

Comment:
I really would prefer not doing this enum change, having all the opcodes listed one-by-one makes it much easier to survey how they're all handed

Reply:
Reverted, although that feels more like a formatting issue that nothing else.